### PR TITLE
AWS - Ensure CIDR modification is documented

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zack Kayyali <zkayyali812@gmail.com>

### CODE REVIEW

**Review by ianf77 — APPROVED** [review]
LGTM. Although I haven't seen these new files.

File: stories/assembly-aap-aws-install.adoc [hunk 1]
```diff
@@ -8,4 +8,6 @@ ifdef::context[:parent-context: {context}]
 include::topics/ref-aap-aws-install-prerequisites.adoc[leveloffset=+1]
 include::topics/proc-aws-create-ec2-pair.adoc[leveloffset=+1]
 include::topics/proc-aws-marketplace.adoc[leveloffset=+1]
+include::topics/proc-aap-aws-modify-alb-cidr-range.adoc[leveloffset=+1]
+include::topics/proc-aap-aws-modify-cidr-ranges.adoc[leveloffset=+1]
```
Comment (ianf77) [inline stories/assembly-aap-aws-install.adoc:12]: Please have these two files reviewed before you add them to the published document.

File: stories/topics/proc-aap-aws-modify-alb-cidr-range.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 = Modify Load Balancers CIDR Range
 
 == Default Load Balancers CIDR Range Configuration on Ansible Automation Platform
-The default CIDR range on Load Balancers are set to 0.0.0.0/0, allowing everyone to access the Load Balancers. If you are looking to update these CIDR range, you can update them before or after the deployment. You can follow the steps below to get this done.
+The default CIDR range on Load Balancers are set to 0.0.0.0/0, allowing everyone to access the Load Balancers. If you are looking to update these CIDR ranges, you can choose to update them either before or after the deployment. You can follow the steps below to get this done.
```
Comment (ianf77) [inline stories/topics/proc-aap-aws-modify-alb-cidr-range.adoc:6]: If you intend to update

File: stories/topics/proc-aap-aws-modify-alb-cidr-range.adoc [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
 
 === Procedure - update before the deployment
 . In the AWS Console, navigate to *CloudFormation* and click on your Red Hat Ansible Automation Platform deployment.
```
Comment (ianf77) [inline stories/topics/proc-aap-aws-modify-alb-cidr-range.adoc:9]: Change line 8 as follows.

===Updating before the deployment

.Procedure

File: stories/topics/proc-aap-aws-modify-cidr-ranges.adoc [hunk 4]
```diff
@@ -3,7 +3,7 @@
...
+If you are looking to modify your CIDR range, this must be done prior to installing. The default CIDR range on your Red Hat AAP on AWS deployment is set to 192.168.0.0/16. This CIDR block is further segmented into 4 subnets (2 private and 2 public) which respectively have CIDR ranges of 192.168.0.0/24, 192.168.1.0/24, 192.168.2.0/24, 192.168.3.0/24. Every subnet has 247 IPv4 address allocated to it and, when creating new components, an IP address from the appropriate subnet is assigned to the new component. It is important to note that these CIDR ranges are immutable post deployment. If you are looking to update your CIDR ranges, your current deployment needs to be deleted and the Red Hat AAP on AWS Cloud Formation Template needs to be updated with new CIDR ranges. You can follow the steps below to get this done.
```
Comment (ianf77) [inline stories/topics/proc-aap-aws-modify-cidr-ranges.adoc:6]: You must modify your CIDR range before installation. 

This CIDR block is further divided into 4 subnets (2 private and 2 public, which have the following CIDR ranges:
.Private
* 192.168.0.0/24
* 192.168.1.0/24
.Public
* 192.168.2.0/24
* 192.168.3.0/24

[IMPORTANT]
=====
These CIDR ranges cannot be changed following deployment.
===== 

If you want to update your CIDR ranges, the current deployment must be deleted and the Red Hat {PlatformName} on AWS Cloud Formation template must be updated with your new CIDR ranges.

The following procedure describes how to do this:

**Review by ianf77 — COMMENTED** [review]
A few comments.

File: stories/topics/proc-aap-aws-modify-cidr-ranges.adoc [hunk 5]
```diff
@@ -3,7 +3,25 @@
...
+
+.Private
+192.168.0.0/24
+192.168.1.0/24
+
+.Public
+192.168.2.0/24
+192.168.3.0/24
+
+IMPORTANT
```
Comment (ianf77) [inline stories/topics/proc-aap-aws-modify-cidr-ranges.adoc:18]: If I put the correct formatting in a comment it does strange things.
Enclose IMPORTANT is square brackets [  ]
Follow it with 
4 equals signs

After the statement, another four equals signs (====)

Comment (zkayyali812) [reply stories/topics/proc-aap-aws-modify-cidr-ranges.adoc]: Updated!

File: stories/topics/proc-aap-aws-modify-cidr-ranges.adoc (hunk 4 above)
Comment (zkayyali812) [reply stories/topics/proc-aap-aws-modify-cidr-ranges.adoc]: I modified the first sentence to be - 
```
If you would like to modify your CIDR range, you must modify your CIDR range before installation.
```

File: stories/topics/proc-aap-aws-modify-alb-cidr-range.adoc (hunk 3 above)
Comment (zkayyali812) [reply stories/topics/proc-aap-aws-modify-alb-cidr-range.adoc:12]: Resolved!

**Comment by ianf77** [comment]
Do you  want this merged?

**Comment by zkayyali812** [comment]
@ianf77 Yes please

**Pull request merged**